aviary/models/test_aircraft/aircraft_for_bench_FwGm.csv

```diff
@@ -2,6 +2,7 @@ aircraft:controls:cockpit_control_mass_scaler,1,unitless
 aircraft:controls:control_mass_increment,0,lbm
```

Comment:
It looks like some recent changes got reverted with this file. Might be from resolving the conflicts.